plugins/outputs/cloudwatch/cloudwatch.go

```
@@ -441,6 +441,11 @@ func (c *CloudWatch) BuildMetricDatum(metric *aggregationDatum) (cloudwatch.Enti
 			continue
 		}
 		if len(distList) == 0 {
+			if metric.Value == nil {
+				log.Printf("E! metric (%s) has nil value, dropping it", *metric.MetricName)
```

can we make this a debug statement? seems like this is a valid otlp payload but we should just silently drop it